cmd/titus-storage/lustre.go

```diff
@@ -54,7 +54,8 @@ func lustreStart(ctx context.Context, config MountConfig) error {
 }
 
 func setupLustreMount(parentCtx context.Context, taskID, cname string, vol *corev1.Volume, vm *corev1.VolumeMount) error {
-	baseMountOptions := []string{"noatime,flock"}
+	l := logger.GetLogger(parentCtx)
+	mountOptions := "recovery_time_soft=300,recovery_time_hard=900"
```

Comment:
It looks like this is constant now, could just pass this directly as `MOUNT_OPTIONS=recov[...]` at L80?